### PR TITLE
[OSCI]Added my name in osci author Prerana

### PR DESCRIPTION
**Pull request by prerana30**
### Description
Adds prerana as a OSCI author to repository.
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by vchrombie** [comment]
Hey @prerana30, the commit doesn't have it, you can amend the signature by `git commit --amend -s`. And then you can force push it using `git push -f origin chore/prerana-pc-author`.

This should help.

**Comment by prerana30** [comment]
> Hey @prerana30, the commit doesn't have it, you can amend the signature by `git commit --amend -s`. And then you can force push it using `git push -f origin chore/prerana-pc-author`.
> 
> This should help.
Thank you it does help.

File: _authors/osci_2023/prerana.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Prerana Sapkota
+short_name:Prerana
+twitter: <twitter handle without the @>
```
Comment (ashwin-pc) [inline _authors/osci_2023/prerana.markdown:4]: Can you remove this line if you are not using it?

**Review by ashwin-pc — CHANGES_REQUESTED** [review]

**Review by dtaivpp — APPROVED** [review]
Looks great!

**Pull request merged**